### DISCUSSION TITLE
Welcome to meent Discussions!

### DISCUSSION THREAD

**Discussion by yonghakim** [kc-ml2/meent#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OwlyWizard** [comment]
Hi, I am a french researcher, working on the design of metasurfaces! Mostly using inverse design techniques (gradient, ML) for routeurs. I'd like to use Meent because of the torch compatibility, and also the dual representation which is rather convenient !

**Reply by yonghakim** [reply]
Thanks and welcome :)
